### ISSUE TITLE
[Request] Crosshairs size/color adjustment

### ISSUE THREAD

**Issue by thepulserifle** [MiSTer-devel/NES_MiSTer#167]
While doing a recent playthrough of The Lone Ranger, I found the crosshairs to be nearly invisible during some scenes due to the size and color disparity.

Currently, the crosshairs change color to blue (from red) on the edges of the screen. Would it be possible to invert this color scheme (or choose your own) in order to be more visible in busier backgrounds?

Can the crosshairs be elongated to enable greater visibility?

**Comment by sorgelig** [comment]
On real NES there is no crosshair at all. All gun games on NES are supposed to be played without any mark.

**Comment by thepulserifle** [comment]
Agreed. If users are able to afford/obtain SNAC and have access to a CRT,
they have the ability to play as intended.

My request is for improvements to the Mouse functionality as it exists.
Quality-of-life improvement. There does not exist a 1:1 “crosshairs”
standard for Zapper emulation.

I did not mean to imply that such a standard existed.
